execution/executor/src/tests/mod.rs

```diff
@@ -357,7 +374,11 @@ fn create_transaction_chunks(
         let txn = encode_mint_transaction(gen_address(i), 100);
         txns.push(txn.into());
     }
-    if BLOCK_GAS_LIMIT.is_none() {
+    if BLOCK_EXECUTOR_ONCHAIN_CONFIG
+        .block_gas_limit_type
+        .block_gas_limit()
```

Comment:
nit: can use a short cut method..

Reply:
I actually think all such accesses should be removed (and we shouldn't have StateCheckpoint depend on it), so this long form is something that should go away

Reply:
actually created a shortcut, so I can more easily know all callers needed to be removed later.